packages/telegram-cloud-storage-stamper/src/__tests__/stamp-test.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from "@jest/globals";
+import TelegramCloudStorageStamper from "../index";
+import { readFixture } from "../__fixtures__/shared";
+import { assertValidSignature } from "./shared";
+
+test("uses provided signature to make stamp", async function () {
+  const { privateKey, publicKey, pemPublicKey } = await readFixture();
+
+  // ToDo: init fails since there isn't an actual Telegram environment here
```

Comment:
Is this still relevant? Looks like `create` is succeeding just fine if this test is passing?

Reply:
This test isnt passing just yet, this still needs some work :)